release/pkg/assets_package_controller.go

```diff
@@ -64,27 +66,49 @@ func (r *ReleaseConfig) GetPackagesAssets() ([]Artifact, error) {
 		ProjectPath:       packagesRootPath,
 		SourcedFromBranch: sourcedFromBranch,
 	}
+	// Remove the OS, and Arch for the Helm chart, and modify the tag based off build-tooling helm/push.sh script.
+	// Currently we must Trim leading v of the tag after the :, and add "-helm to the end of tag.
+	// 123456.dkr.ecr.us-west-2.amazonaws.com/eks-anywhere-packages:v0.1.2-2e9994fd1afb2216a51fa474ac5d7dc6a772bb62
+	// >>>
+	// 123456.dkr.ecr.us-west-2.amazonaws.com/eks-anywhere-packages:0.1.2-2e9994fd1afb2216a51fa474ac5d7dc6a772bb62-helm
 	artifacts := []Artifact{{Image: imageArtifact}}
+	helmImageArtifact := &ImageArtifact{
+		AssetName:         packagesHelmChart, // This needs to differ from the image name for later steps
+		SourceImageURI:    sourceImageUri,
+		ReleaseImageURI:   strings.ReplaceAll(fmt.Sprintf("%s-helm", releaseImageUri), "packages:v", "packages:"),
```

Comment:
Wouldn't we use the shasum for the tag?

Reply:
This is looking up the dev private ECR tag. I looked in the account, and the image tags are getting `v${GIT_TAG}-{GIT_HASH)`  and the helm had `${GIT_TAG}-{GIT_HASH)-helm` we just needed to trim the image tag based off the `helm/push.sh` script tagging.